### PR TITLE
OUT 1009 Adjust vertical spacing of text input in comment box

### PR DESCRIPTION
**Pull request by aatbip**
### Changes

- [x] Removes gap from the comment input

### CODE REVIEW

File: src/app/detail/ui/styledComponent.tsx [hunk 1]
```diff
@@ -87,6 +87,10 @@ export const CommentCardContainer = styled(Stack)(({ theme }) => ({
   background: theme.color.base.white,
   padding: '8px',
   width: '100%',
+  //remove gap from the parent div of the Tapwrite component
+  '& > div:first-of-type': {
+    gap: '0px !important',
+  },
```
Comment (priosshrsth) [inline src/app/detail/ui/styledComponent.tsx:93]: Can we not update tapwrite to accept gap props, same as padding and width? or better yet style props?

Comment (aatbip) [reply src/app/detail/ui/styledComponent.tsx:93]: Added a PR in tapwrite repo - https://github.com/aatbip/tapwrite/pull/16

Comment (priosshrsth) [reply src/app/detail/ui/styledComponent.tsx]: I guess you need to add new version of tapwrite then, don't you?

**Review by rrojan — APPROVED** [review]
LGTM on Tapwrite changes :trophy:

**Pull request merged**